utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -38,7 +38,11 @@ class HtmlParser private constructor(
     }
 
     if (htmlContent.contains(CUSTOM_IMG_TAG)) {
-      htmlContent = htmlContent.replace(CUSTOM_IMG_TAG, REPLACE_IMG_TAG, /* ignoreCase= */false)
+      htmlContent = htmlContent.replace(
+        CUSTOM_IMG_TAG,
+        REPLACE_IMG_TAG,
+        /* ignoreCase= */false
```

Comment:
Should we add space in between `/* ignoreCase= */` and `false`?

Reply:
Done